### PR TITLE
Don't signout on no certificate translated exit page and updated content

### PR DESCRIPTION
**Pull request by tchow8**
# Description

[DIV-4108](https://tools.hmcts.net/jira/browse/DIV-4108)

This PR disables destroying the session and sign out on reaching the no translated certificate exit page.
The content on the exit page is also updated.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/steps/check-your-answers/index.js [hunk 1]
```diff
@@ -211,7 +212,7 @@ module.exports = class CheckYourAnswers extends ValidationStep {
       delete session.nextStepUrl;
     }
 
-    if (nextStep && nextStep !== this && !(nextStep instanceof DestroySessionStep)) {
+    if (nextStep && nextStep !== this && !(nextStep instanceof DestroySessionStep) && !(nextStep instanceof ExitStep)) {
```
Comment (dtaylor7) [inline app/steps/check-your-answers/index.js]: can we break this logic out as its becoming unwieldy e.g.
```
const nextStepIsCYAStep =  nextStep == this;
const nextStepIsDestroyStep = nextStep instanceof DestroySessionStep;
const nextStepIsExitStep = nextStep instanceof ExitStep;
if (nextStep && !nextStepIsCYAStep && !nextStepIsDestroyStep && !nextStepIsExitStep) {
```

File: app/core/steps/ExitStep.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const Step = require('./Step');
+
+module.exports = class ExitStep extends Step {
```
Comment (dtaylor7) [inline app/core/steps/ExitStep.js]: will these be confusing for future devs, the fact we have destroy session step, exit step ( which does nothing ).... page we should just extend step on the `ExitNoCertificateTranslated`

File: app/steps/check-your-answers/index.js (hunk 1 above)
Comment (dtaylor7) [reply app/steps/check-your-answers/index.js]: dont remember logic, but presume it `ExitNoCertificateTranslated` found wont the next step be undefined anyway? whats the purpose of adding this to the if statement?

**Comment by nsingamsetty** [comment]
This is tested and working fine.Will merge this PR on behalf of Phi - Please refer DIV-4108 for evidences.

File: app/steps/check-your-answers/index.js (hunk 1 above)
Comment (tchow8) [reply app/steps/check-your-answers/index.js]: It the step before that sets the nextStep to ExitNoCertificateTranslated. So when they resume, and its not a DestroySessionStep, they resume on the Exit page, when they want to resume on the previous page.

File: app/core/steps/ExitStep.js (hunk 2 above)
Comment (tchow8) [reply app/core/steps/ExitStep.js]: Makes sense, I was thinking of doing that but I felt I would be duplicating code across every future exit step. But if you feel that is better, I can modify that. Or would renaming this step to like TerminalStep or something make more sense?

**Comment by nsingamsetty** [comment]
This is tested and signed off. Merging this on behalf of Phi.

**Pull request merged**